index.html

```diff
@@ -1,160 +1,279 @@
+                ><img src="./image/twitter_icon.png" alt="twitter_icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#" class="about_icon"
+                ><img src="./image/m_icon.png" alt="m_icon"
+              /></a>
+            </li>
+          </ul>
+    
+          <a href="#" class="button2">Get my resume</a>
+        </div>       
+        <ul class="about_group">
+          <li>
+            <div class="lang_tech">
+              <h3 class="lang_title">Languages</h3>
+              <img class="lang_arrow" src="./image/Disabled.png" alt="" />
+            </div>
+  
+            <ul class="about_flex">
+              <li class="about_skill">
+                <img class="about_img" src="./image/JS_icon.png" alt="" />
+                <h4 class="about_lang">JavaScript</h4>
+              </li>
+  
+              <li class="about_skill">
+                <img class="about_img" src="./image/html_icon.png" alt="" />
+                <h4 class="about_lang">HTML</h4>
+              </li>
+  
+              <li class="about_skill">
+                <img class="about_img" src="./image/css_icon.png" alt="" />
+                <h4 class="about_lang">CSS</h4>
+              </li>
+            </ul>
+          </li>
+  
+          <li class="lang_tech border">
+            <h3 class="lang_title">Frameworks</h3>
+            <img class="lang_arrow" src="./image/narrow.png" alt="" />
+          </li>
+  
+          <li class="lang_tech border">
+            <h3 class="lang_title">Skills</h3>
+            <img class="lang_arrow" src="./image/narrow.png" alt="" />
+          </li>
         </ul>
+      </section>
+  
+      <section class="sectionForm">
+        <div class="form">
+            <h2 class="form-title">Contact me</h2>
+            <p class="form-parragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+          <form class="form-form" action="https://formspree.io/f/xlevbkay" method="post">
+            <ul class="ul-form">
+              <input class="formInput"
+                type="text"
+                name="name"
+                placeholder="   Alexandra Loremipsum"
+                maxlength="30"
+                required
+                />
+              <input class="formInput"
+                  type="email"
+                  name="email"
+                  placeholder="   alexandra@gmail.com"
+                  required
+              />
+              <textarea class="formArea"
+                  type="text"
+                  name="message"
+                  rows="5"
+                  maxlength=""
+                  placeholder="   Write your message here"
+                  required
+              ></textarea>
 
-    </section>
-</body>
-</html>
+            </ul>
+            <button class="button-form" type="submit">Get in touch</button>
 
+          </form>
```

Comment:
- [ OPTIONAL ] For the form section you indeed made the changes but I noticed the background image is just blue in your design but there is an background image in the orignal design

- Your design
![image](https://user-images.githubusercontent.com/58553711/193432767-095140b5-61d0-46e2-a5a2-9eb7d4677a26.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/193432774-837942c4-9b1a-4314-beb2-3bf69bebd5d8.png)